lib/util/cache/repository/impl/memory.ts

```diff
@@ -0,0 +1,19 @@
+import type { RepoCache, RepoCacheData } from '../types';
+
+export class MemoryRepoCache implements RepoCache {
```

Comment:
This essentially performs the same job that `RepoBaseCache` did when not sub-classed.  I considered naming this `NullRepoCache` to indicate that it's not intended for caching, but the original behavior was to act like an in-memory cache, thus the name here.